.eslintrc.yaml

```diff
@@ -0,0 +1,85 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
```

Comment:
Can you update the comments to include what modifications we have for this specific project?